runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaTask.kt

```diff
@@ -193,12 +194,25 @@ open class DokkaTask : DefaultTask(), Configurable {
             return null
         }
 
+        val sourceSetIdCache: Map<String, String> = defaultModulesConfiguration.map {
```

Comment:
I think we can find a more straight-forward way here 🤔

Reply:
What do you mean?